pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -676,6 +676,7 @@ func (c *Reconciler) createTaskRun(ctx context.Context, rprt *resources.Resolved
 		tr.Spec.TaskRef = &v1beta1.TaskRef{
 			Name: rprt.ResolvedTaskResources.TaskName,
 			Kind: rprt.ResolvedTaskResources.Kind,
+			//TODO: Add TaskRef metadata?
```

Comment:
I don't know what is the expected behavior of this block, and since it's already missing the `APIVersion` field, I left it unset by now.